torchbenchmark/util/model.py

```diff
@@ -164,7 +164,7 @@ def __post__init__(self):
                 self.dargs.precision == "fp16"
                 or self.dargs.precision == "amp"
                 or (self.dynamo and self.opt_args.torchdynamo == "fx2trt")
-                or (not self.dynamo and self.opt_args.fx2trt)
+                or (not self.dynamo and (self.device == "cuda" and self.opt_args.fx2trt))
```

Comment:
It should be:

```suggestion
                or (not self.dynamo and (self.device == "cuda" and self.opt_args.backend == "fx2trt"))
```

Reply:
thanks for the quick review, will update.
yes, looks like the recent cuda fx2trt changes introduced this.